docs/sql-ref-syntax-qry-window.md

```diff
@@ -19,4 +19,208 @@ license: |
   limitations under the License.
 ---
 
-**This page is under construction**
+### Description
+
+Window functions operate on a group of rows, referred to as a window, and calculate an aggregated value for each row based on the specified window. Window functions are useful for processing tasks such as calculating a moving average, computing a cumulative, or accessing the value of rows given the relative position of the current row. Spark SQL supports three types of window functions:
+
```

Comment:
Does this look better now? @maropu @viirya

Reply:
Also cc @srowen 
Please feel free to rephrase. Thanks!

Reply:
computing a cumulative -> computing a cumulative _sum_ (or anything similar: average, statistic)

Reply:
Thanks, it looks better. How about putting the last statement in a new line?; 
```
...the current row. 

Spark SQL supports three types of window functions:

  * Ranking Functions
  * Analytic Functions
  * Aggregate Functions
```

Reply:
We need this list here? The `Syntax` section has the same list.